### PR TITLE
Solarus Game Engine

### PR DESCRIPTION
**Pull request by santiagoitzcoatl**
It's my first dash Docset, I hope I'm doing everything fine.

Solarus is a free and open-source Action-RPG game engine and has an increasing community of developers and users.

### CODE REVIEW

File: docsets/Solarus/docset.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Solarus",
+    "version": "1.6.0",
+    "archive": "Solarus.docset",
```
Comment (Kapeli) [inline docsets/Solarus/docset.json]: This needs to be a `tgz` and it needs to exist. It's not enough to have the archive only in `versions`.

**Comment by Kapeli** [comment]
You've got a `solarus` folder and a `Solarus` folder. You need to delete one of them.

File: docsets/Solarus/docset.json (hunk 1 above)
Comment (santiagoitzcoatl) [reply docsets/Solarus/docset.json]: hopefully, it's fine now, thanks.

**Comment by santiagoitzcoatl** [comment]
Thank you and sorry about the mess.

**Pull request merged**